test/test_ldappool.py

```
@@ -0,0 +1,291 @@
+        max=1,
+        params={"prewarm": False, "lock_timeout": 3},
+    )
+    oiter = len(outage)
+    for _ in range(oiter):
+        with pytest.raises((ldap.SERVER_DOWN, LDAPPoolExhausted)):
+            conn = pool.get()
+            conn.authenticate("uid=jdoe,ou=users,dc=example,dc=com", "changeme")
+    for _ in range(oiter):
+        try:
+            conn = pool.get()
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)